doc/JSON_INFO.md

```diff
@@ -3178,6 +3181,9 @@ Weakpoints only match if they share the same id, so it's important to define the
 "price_postapoc": "1 USD",                       // Same as price but represent value post cataclysm. Can use string "cent" "USD" or "kUSD".
 "stackable": true,                           // This item can be stacked together, similarly to `charges`
 "degradation_multiplier": 0.8,               // Controls how quickly an item degrades when taking damage. 0 = no degradation. Defaults to 1.0.
+"solar_efficiency": 0.3,                     // Efficiency of solar energy conversion for solarpacks; require SOLARPACK_ON to generate electricity; default 0
+"source_monster": "mon_child",               // This item is corpse of this monster (so it has weight and volume of this monster), and revive into this monster; require COPRSE flag
```

Comment:
So does mon_child revive into a mon_zombie_child?

Reply:
oh, i copied the wrong number, i checked out from #68390, it has this changes actually - this one has only small code stuff and MA doc updates
answering your question - no, it actually revive into `mon_child`, <https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/monsters/mammal.json#L14>, which already zombifies into `mon_zombie_child`; i'll change this monster into some another in that PR